packages/unleash-api/lib/db/event.js

```diff
@@ -21,7 +22,7 @@ module.exports = function(db) {
         return db
         .select(EVENT_COLUMNS)
         .from('events')
-        .whereRaw("data ->> 'name' = ?", [name])
+        .whereRaw('data ->> \'name\' = ?', [name])
```

Comment:
I can manually fix it, will have to replace \' with \", thats what you want right?